<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-16T20:40:57 IST" name="All Test Suite" finished-at="2023-01-16T20:41:31 IST" duration-ms="34100">
    <groups>
    </groups>
    <test started-at="2023-01-16T20:40:57 IST" name="/Users/k0d0866/IdeaProjects/Integration_testing/src/test/java/TestSuite/Login.java" finished-at="2023-01-16T20:41:31 IST" duration-ms="34100">
      <class name="TestSuite.Login">
        <test-method signature="appLaunch()[pri:1, instance:TestSuite.Login@63a65a25]" started-at="2023-01-16T20:40:57 IST" name="appLaunch" finished-at="2023-01-16T20:41:14 IST" duration-ms="17469" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appLaunch -->
        <test-method signature="loginValidation()[pri:2, instance:TestSuite.Login@63a65a25]" started-at="2023-01-16T20:41:14 IST" name="loginValidation" finished-at="2023-01-16T20:41:31 IST" duration-ms="16527" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hello, Kiran Kumar Dash] but found [null]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hello, Kiran Kumar Dash] but found [null]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at TestSuite.Login.loginValidation(Login.java:35)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginValidation -->
        <test-method is-config="true" signature="aftertest()[pri:0, instance:TestSuite.Login@63a65a25]" started-at="2023-01-16T20:41:31 IST" name="aftertest" finished-at="2023-01-16T20:41:31 IST" duration-ms="77" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftertest -->
      </class> <!-- TestSuite.Login -->
    </test> <!-- /Users/k0d0866/IdeaProjects/Integration_testing/src/test/java/TestSuite/Login.java -->
  </suite> <!-- All Test Suite -->
</testng-results>
